packages/service-integration/src/service_integration/commands/compose.py

```diff
@@ -0,0 +1,134 @@
+from contextlib import suppress
+from pathlib import Path
+from typing import Dict, List
+
+import click
+import yaml
+
+from ..compose_spec_model import ComposeSpecification
+from ..labels_annotations import to_labels
+from ..oci_image_spec import LS_LABEL_PREFIX, OCI_LABEL_PREFIX
+from ..osparc_config import MetaConfig, RuntimeConfig
+from ..osparc_image_specs import create_image_spec
+
+
+def create_docker_compose_image_spec(
+    io_config_path: Path,
+    service_config_path: Path = None,
+) -> ComposeSpecification:
+    """Creates image compose-spec"""
+
+    config_basedir = io_config_path.parent
+
+    # required
+    meta_cfg = MetaConfig.from_yaml(io_config_path)
+
+    # optional
+    runtime_cfg = None
+    if service_config_path:
+        with suppress(FileNotFoundError):
+            # TODO: should include default?
+            runtime_cfg = RuntimeConfig.from_yaml(service_config_path)
```

Comment:
Here the idea is that ``service_config_path`` is *optional* 

I agree with you that *the intent* was not nicely coded . I will redo it.
I also agree that at the CLI level we could echo the user something like "since you did not specified a runtime config, we use the default one ... or so"